test/e2e/test_tools.py

```diff
@@ -155,14 +155,6 @@ def test_excel_to_json_list(self) -> None:
                         excelfolder='testdata/lists',
                         outfile='testdata/tmp/_lists-out.json')
 
-    def test_excel_to_json_resources(self) -> None:
-        resources_excel2json(excelfile='testdata/Resources.xlsx',
-                             outfile='testdata/tmp/_resources-out.json')
-
-    def test_excel_to_json_properties(self) -> None:
-        properties_excel2json(excelfile='testdata/Properties.xlsx',
-                              outfile='testdata/tmp/_properties-out.json')
-
```

Comment:
Good point. I restored them, but deleted `test_validate_ontology()`, because it is not a dsp-tools command, but already contained in the `create` command